pkgs/applications/editors/vim/plugins/update.py

```diff
@@ -1,181 +1,210 @@
-#!/usr/bin/env python
-
-# run with:
-# $ nix run .\#vimPluginsUpdater
-# format:
-# $ nix run nixpkgs#python3Packages.ruff -- update.py
-# type-check:
-# $ nix run nixpkgs#python3Packages.mypy -- update.py
-# linted:
-# $ nix run nixpkgs#python3Packages.flake8 -- --ignore E501,E265,E402 update.py
-
-# If you see `HTTP Error 429: too many requests` errors while running this
-# script, refer to:
-#
-# https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/vim.section.md#updating-plugins-in-nixpkgs-updating-plugins-in-nixpkgs
-#
-# (or the equivalent file /doc/languages-frameworks/vim.section.md
-# from Nixpkgs master tree).
-#
-
-import inspect
-import json
-import logging
+from __future__ import annotations  # noqa: INP001
```

Comment:
```suggestion
#!/usr/bin/env nix-shell
#!nix-shell -i python3 -p "python3.withPackages(ps: with ps; [ nixpkgs-updaters-library ])"

from __future__ import annotations  # noqa: INP001
```

Reply:
This will let you invoke it directly. 
![image](https://github.com/user-attachments/assets/6ebf4f45-704c-4a72-998d-67249326f7ea)

Reply:
Good idea!